Library/Formula/wireshark.rb

```diff
@@ -20,14 +16,6 @@ class Wireshark < Formula
     sha256 "7980680d82cb4c3bf01c6b8c1f3970767773d9a09d5573e3ba418fe08f224bb4" => :mavericks
   end
 
-  devel do
-    url "https://www.wireshark.org/download/src/all-versions/wireshark-2.0.0rc3.tar.bz2"
-    mirror "https://1.eu.dl.wireshark.org/src/wireshark-2.0.0rc3.tar.bz2"
-    sha256 "11f46b2d202f923c87a0b8e1a07bd9910bbcc5c265c69de3a23dde82e0f647e3"
-
-    depends_on "homebrew/dupes/libpcap" if MacOS.version == :mavericks
```

Comment:
This line is now required for all builds as `Mavericks` ships with an older version, [1.3.0](http://opensource.apple.com/release/os-x-1095/), of `libpcap`.